<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_team</class>
 <widget class="QDialog" name="display_team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="styleSheet">
              <string notr="true">background-color: white</string>
             </property>
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QCheckBox" name="team_name_checkBox">
                <property name="text">
                 <string>Team</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="arena_checkBox">
                <property name="text">
                 <string>Arena</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="capacity_checkBox">
                <property name="text">
                 <string>Capacity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="year_checkBox">
                <property name="text">
                 <string>Year Joined</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="coach_checkBox">
                <property name="text">
                 <string>Coach</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_sort">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: white</string>
               </property>
               <property name="text">
                <string>Sort By:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="combo_sort">
               <item>
                <property name="text">
                 <string>Team Name</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Arena Name</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Capacity</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Year Joined</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_sort_2">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: white</string>
                 </property>
                 <property name="text">
                  <string>Filter By:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="eastern_conf_Button">
                 <property name="text">
                  <string>Eastern Conference</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="western_conf_Button">
                 <property name="text">
                  <string>Western Conference </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="division_comboBox">
               <item>
                <property name="text">
                 <string>Atlantic</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Central</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Southeast</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Northwest</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Pacific</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Southwest</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="Line" name="line">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_team">
                 <item>
                  <property name="text">
                   <string>Select Team</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="push_souvenirs">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Souvenirs</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>546</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="reload_button">
         <property name="text">
          <string>Reload Table</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelTeamS">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_logo">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/resources/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
